pkg/lv_drivers/Makefile

```
@@ -0,0 +1,18 @@
+PKG_NAME=lv_drivers
+PKG_URL=git@github.com:lvgl/lv_drivers.git
```

```suggestion
PKG_URL=https://github.com/lvgl/lv_drivers.git
```
otherwise this requires an authenticated access to github and can cause problems sometimes.